### DISCUSSION TITLE
Everyday Usage

### DISCUSSION THREAD

**Discussion by zany2dmax** [Homebrew/discussions#5487] [Everyday usage] [answered]
### Output of `brew config`

```shell
└─[0] brew config
HOMEBREW_VERSION: 4.3.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 1c98a33a376e5b35ebc801d0f7d6ba7b9eb118ac
Last commit: 4 days ago
Core tap JSON: 11 Jul 17:12 UTC
Core cask tap JSON: 11 Jul 17:12 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
ftp_proxy: http://localhost:9000
http_proxy: http://localhost:9000
https_proxy: http://localhost:9000
no_proxy: localhost,127.0.0.1,.comerica.net
Homebrew Ruby: 3.3.3 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
Rosetta 2: false
┌─[jxl078@PK2HXHX7D6] - [~] - [2024-07-11 03:07:47]
└─[0] 

```

### Output of `brew doctor`

```shell
└─[0] brew doctor
Your system is ready to brew.
```

### Description of issue

Hi, my MacBook Pro (M1 Pro) is owned by my employer and doessn't allow me admin (sudo) rights.  I was able to get homebrew installed as an exception by a workstation, but there are certain formulae that try and install too the system level /Applications that I do not have permissions (nor will get permissions to). I have been able to install other apps to my Users /Applications directory and they work OK for the most part.

is there any mechanism in the brew CLI for me to force the output to that directory instead of the system level one?  Example for zed below. 

┌─[jxl078@PK2HXHX7D6] - [~] - [2024-07-11 01:11:51]
└─[1] brew install zed
==> Downloading https://formulae.brew.sh/api/formula.jws.json
######################################################################### 100.0%
==> Downloading https://formulae.brew.sh/api/cask.jws.json
######################################################################### 100.0%
==> Downloading https://zed.dev/api/releases/stable/0.143.6/Zed-aarch64.dmg
==> Downloading from https://objects.githubusercontent.com/github-production-rel
######################################################################### 100.0%
==> Installing Cask zed
==> Moving App 'Zed.app' to '/Applications/Zed.app'
Password:

**Comment by SMillerDev** [comment, marked as answer]
https://docs.brew.sh/Manpage#global-cask-options

**Reply by zany2dmax** [reply]
Ah I missed that flag!  My bad I did 'man brew' looking for it.

--appdir
Target location for Applications (default: /Applications).
